cinema/migrations/0003_movie_duration.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('cinema', '0002_initial'),
+        ("cinema", "0002_initial"),
     ]
 
     operations = [
         migrations.AddField(
-            model_name='movie',
-            name='duration',
+            model_name="movie",
+            name="duration",
             field=models.IntegerField(default=123),
             preserve_default=False,
```

Comment:
The `duration` field is added with a default value of 123, but `preserve_default` is set to `False`. This means the default value will not be preserved in the database schema. If there are existing records, this might cause issues unless all records are updated with a valid duration. Consider setting `preserve_default=True` if you want to ensure the default value is applied to existing records.